docs/guides/Styling the UI.md

```diff
@@ -0,0 +1,74 @@
+# Styling the UI
+
+If you're using [`NavigationMapViewController`](https://www.mapbox.com/mapbox-navigation-ios/navigation/0.18.1/Classes/NavigationViewController.html), it's possible to apply custom fonts and colors to various parts of the UI.
```

Comment:
There’s no need to link this symbol. As long as you surround it in backticks, jazzy will automatically link it to the correct documentation in the same version of the docset.

Reply:
Neat